.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove any files with local settings (editor settings etc.) from the repo. You can keep a local copy and add `.vscode/settings.json` to `.gitignore`.